nuspec/nuget/GitReleaseManager.nuspec

```diff
@@ -11,8 +11,8 @@
     <repository type="git" url="https://github.com/GitTools/GitReleaseManager.git"/>
     <icon>package_icon.png</icon>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Tool for creating and exporting releases for software applications hosted on GitHub</description>
-    <summary>Tool for creating and exporting releases for software applications hosted on GitHub</summary>
+    <description>Tool for creating and exporting releases for software applications</description>

```

Comment:
instead of removing `hosted on GitHub`, maybe change it to `hosted on a remote provider`?

Reply:
Done